src/google/protobuf/extension_set.h

```diff
@@ -419,13 +419,8 @@ class LIBPROTOBUF_EXPORT ExtensionSet {
   // are never forced to serialize deterministically.
   uint8* SerializeWithCachedSizesToArray(int start_field_number,
                                          int end_field_number,
-                                         uint8* target) const {
```

Comment:
Was the problem that MSVC doesn't like it if you have a member function defined in the .h file calling another member function in the .cc file?